libraries/AP_MSP/AP_MSP_Telem_DJI.cpp

```diff
@@ -108,7 +111,7 @@ MSPCommandResult AP_MSP_Telem_DJI::msp_process_out_esc_sensor_data(sbuf_t *dst)
 {
 #if HAL_WITH_ESC_TELEM
     const auto msp = AP::msp();
-    if (msp && (msp->_options & AP_MSP::OPTION_TELEMETRY_DJI_WORKAROUNDS)) {
+    if (msp && msp->check_option(AP_MSP::Option::TELEMETRY_DJI_WORKAROUNDS)) {
```

Comment:
```suggestion
    if (msp && msp->option_enabled(AP_MSP::Option::TELEMETRY_DJI_WORKAROUNDS)) {
```

would seem clearer.  pre-existing method, 'though

Reply:
I grepped the source tree and there are no other option_enabled(), there are though a couple of check_option in the AP_OSD library, I guess I took inspiration by those but no big deal, should we have some strict naming rules for such things?